### PR TITLE
kubemacpool: Add stable branch release-0.45 lane

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
This PR adds a stable branch [release-0.45](https://github.com/k8snetworkplumbingwg/kubemacpool/tree/release-0.45) lane for kubemacpool.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.45.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  k8snetworkplumbingwg/kubemacpool:
+    - name: pull-kubemacpool-unit-test-v0.45
+      context: pull-kubemacpool-unit-test
+      branches:
+        - release-0.45
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
```
[the remark this answers is not in the document]
Comment (RamLavi) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.45.yaml]: [DONE](https://github.com/kubevirt/project-infra/compare/6f6bd24d8d6166c524f82de816cb6f208265e97f..14809dab5b9decd94577bd6e206622f13c7429a2)

**Comment by oshoval** [comment]
"/skip-review" didnt know this one, what it does ? 
you can just lgtm approve isnt it ?

**Comment by RamLavi** [comment]
> "/skip-review" didnt know this one, what it does ? you can just lgtm approve isnt it ? so might not related

tide was stuck on this label missing, tried to see if I can add the missing label.. I can't :)

**Pull request merged**